CorsixTH/Src/th_movie.h

```diff
@@ -411,22 +423,22 @@ class movie_player {
 #endif
 
   int audio_buffer_size;      ///< The current size of audio data in
-                              ///< #m_pbAudioBuffer
+                              ///< #audio_buffer
   int audio_buffer_index;     ///< The current position for writing in
-                              ///< #m_pbAudioBuffer
-  int audio_buffer_max_size;  ///< The capacity of #m_pbAudioBuffer (allocated
+                              ///< #audio_buffer
+  int audio_buffer_max_size;  ///< The capacity of #audio_buffer (allocated
                               ///< size)
   uint8_t* audio_buffer;      ///< An audio buffer for playback
```

Comment:
an class with data storage and these administrative numbers could be useful?

Reply:
std::array/std::vector can be used for that; will get around to replacing these in this patch set.